nginx/containerbuddy.json

```diff
@@ -1,5 +1,5 @@
 {
-  "consul": "consul:8500",
+  "consul": "{{ .CONSUL }}:8500",
   "onStart": "/opt/containerbuddy/reload.sh",
```

Comment:
Noted in https://github.com/tgross/triton-nginx/issues/6 and https://github.com/tgross/workshop-autopilot/issues/2. We'll need to update in the Docker image and here in the Containerbuddy config.